pytorch_lightning/loggers/wandb.py

```diff
@@ -98,6 +99,13 @@ def __init__(
         if wandb is None:
             raise ImportError('You want to use `wandb` logger which is not installed yet,'  # pragma: no-cover
                               ' install it with `pip install wandb`.')
+
+        if offline and log_model:
+            raise MisconfigurationException(
+                f'log_model={log_model} and offline={offline} is an invalid configuration'
+                ' since model checkpoints cannot be uploaded in offline mode.'
+            )
```

Comment:
```suggestion
            raise MisconfigurationException(
                f'Providing log_model={log_model} and offline={offline} is an invalid configuration'
                ' since model checkpoints cannot be uploaded in offline mode.'
                'Hint: Set offline=False to log your model.'
            )
```